hw/ip/sysrst_ctrl/doc/theory_of_operation.md

```diff
@@ -0,0 +1,174 @@
+# Theory of Operations
```

Comment:
```suggestion
# Theory of Operation
```

According to the new file name and to be aligned with other modules

Reply:
Thanks, this should be updated now.